Microsoft.Toolkit.Uwp.Connectivity/Microsoft.Toolkit.Uwp.Connectivity.csproj

```diff
@@ -18,6 +18,10 @@
 	</ItemGroup>
 
 	<ItemGroup>
+	  <PackageReference Include="Uno.UI" Version="3.0.17" />
+	</ItemGroup>
+
+	<ItemGroup>
```

Comment:
The Uno.UI package is defined above in all projects.

Reply:
Sure i'm going to update to a more recent one.

Reply:
Oups got the mistake 👍